### PR TITLE
docker: add compose and skeleton for Docker

### PR DESCRIPTION
**Pull request by Hjortsberg**
This is a startingpoint for docker. Unsure wether the Dockerfile and docker-composefile should be in an folder, or root directory?

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM golang:1.13-alpine
+LABEL authors="Philip Hjortsberg <philip@hjortsberg.me>, William Wennerström <william@willeponken.me>, Edvin Sladic <edvin@sladic.se>"
```
Comment (optmzr) [inline Dockerfile]: Fy fan vad snygga mejl vi alla har alltså

(jag har bytt till william@optmzr.se btw :eyes:)

**Comment by edvinnn** [comment]
Vad är kommandona för att:

1. Starta upp flera containers?
2. Ansluta till dem?

**Comment by Hjortsberg** [comment]
@optmzr  235 tries later, i think i managed to do it! :champagne: 
I just want confirmation that it looks correct before i merge.

**Comment by Hjortsberg** [comment]
closes #4

**Pull request merged**